docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,133 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS with a Kubernetes master and one or more nodes (minions)
+    - We recommend the [Kubernetes instructions for AWS](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md). This guide was created with the `kube-up` script in mind.
+* SSH access to your Kubernetes master and nodes
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Installing Calico
+On each of your AWS Instances, you will need to download and install the `calicoctl` binary
+```
+wget https://github.com/projectcalico/calico-docker/releases/download/v0.9.0/calicoctl
+chmod +x calicoctl
+sudo mv calicoctl /usr/bin/
+```
+
+## Configuring the Master
+On your master, you will need to setup an etcd directory specifically for Calico. To do so, you will need to download our etcd manifest
+```
+wget https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/master/manifests/calico-etcd.manifest
+```
+
+Replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, place the manifest file in the `/etc/kubernetes/manifests/` directory. 
```

Comment:
Is the hostname actually valid in that file?  I think all instances need to be the master's IP.